.github/workflows/build-pr.yml

```diff
@@ -0,0 +1,23 @@
+name: build-pr
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout source code
+        uses: actions/checkout@master
+      - name: setup go environment
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.16.5'
```

Comment:
might as well stick to 1.15

Reply:
instead, I just updated everything to 1.16 in 01d702f778df4727c21d3e88e723a592197547ab